containers/src/Kokkos_UnorderedMap.hpp

```diff
@@ -767,6 +810,32 @@ class UnorderedMap {
     }
   }
 
+  KOKKOS_FORCEINLINE_FUNCTION
+  bool is_op_noop() const {
```

Comment:
Do we need these functions at all?

Reply:
The main need is for asserting in the unit-test. I am open to alternative approaches, perhaps just in the unit-test. Thoughts? I will try to remove these.